sklearn/utils/tests/test_multiclass.py

```diff
@@ -361,9 +349,14 @@ def test_class_distribution():
 
 
 def test_safe_split_with_precomputed_kernel():
+    from sklearn.model_selection import ShuffleSplit
```

Comment:
Not local imports. That's very unconventional in tests. But it's useful only to load iris here.